src/main/resources/doc/en/html/guide/verify/test.html

```diff
@@ -7,26 +7,26 @@
     <meta http-equiv="content-type" content="text/html; charset=utf-8">
     <meta http-equiv="Content-Language" content="fr">
     <title>
-      Fenêtre Vecteur de test
+      Test Vector Window
     </title>
     <link rel="stylesheet" type="text/css" href="../../style.css">
   </head>
   <body>
     <div class="maindiv">
       <h1>
-        Fenêtre Vecteur de test
+        Test Vector Window
       </h1>
       <p>
-        Depuis la fenêtre Vecteur de test vous pouvez charger un vecteur de test à partir d'un fichier et Logisim lancera des tests sur le circuit actuel. Il n'y a qu'une seule fenêtre de vecteur de test pour le projet et la table changera pour refléter le circuit simulé dans la fenêtre du projet. Notez cependant que le module vecteur de test exécute une copie séparée du simulateur de circuit et n'interfère donc pas avec la simulation dans la fenêtre du projet et n'est pas influencé par cette simulation.
+        From the Test vector window you can load a test vector from a file and Logisim will run tests on the current circuit. There is only one test vector window for the project and the table will change to reflect the simulated circuit in the project window. Note however that the test vector module runs a separate copy of the circuit simulator and therefore does not interfere with the simulation in the project window and is not influenced by this simulation.
       </p>
       <p>
-        Pour l'exemple, nous allons tester le circuit ci-dessous. Ce circuit donne les résultats de cinq fonctions logiques à partir de deux entrées. Il comporte une erreur la porte NAND du bas devrait être une porte AND.
+        For the example, we will test the circuit below. This circuit gives the results of five logic functions from two inputs. It has an error the bottom NAND gate should be an AND gate.
```

Comment:
The last part is not a sentence. I suggest adding "because" to get: "It has an error because the bottom..."

That change should also be made in the other files that contain English.

In fact, all the files that contain English (including en) should be identical as far as I know (except for the line that specifies the Content-Language). Currently, de, ru, and pt are actually in English and are the same. I didn't check if any others exist. Does this PR make en match? No. At least line 20 is different here than in the other files.

In an ideal implementation, we wouldn't have multiple files that have essentially the same content. I recommend we make all the files match for this PR and raise the issue of redundancy in the implementation elsewhere.

Reply:
I'll see what I can do.

Reply:
@dtowell: It would be nice if you could incorporate @davidhutchens's suggestion so that this PR can be merged.